test/integ/test_put_get_user_stage.py

```diff
@@ -39,6 +40,72 @@ def test_put_get_small_data_via_user_stage(is_public_test, tmpdir, conn_cnx, fro
     )
 
 
+@pytest.mark.skipolddriver
+@pytest.mark.aws
+@pytest.mark.parametrize(
+    "from_path",
+    [True, False],
+)
+@pytest.mark.parametrize(
+    "accelerate_config",
+    [True, False],
+)
+def test_put_get_accelerate_user_stage(
+    is_public_test, tmpdir, conn_cnx, from_path, accelerate_config
+):
+    from snowflake.connector.file_transfer_agent import SnowflakeFileTransferAgent
+    from snowflake.connector.s3_storage_client import SnowflakeS3RestClient
+
+    """[s3] Puts and Gets Small Data via User Stage."""
+    if is_public_test or "AWS_ACCESS_KEY_ID" not in os.environ:
```

Comment:
I think you can just [tag the test as an internal test](https://github.com/snowflakedb/snowflake-connector-python/blob/f2f7e6f35a162484328399c8a50a5015825a5573/test/integ/test_cursor_binding.py#L139) to achieve the same thing